VS_Project/Pokerapp/Pokerapp/Timer.cshtml

```diff
@@ -0,0 +1,16 @@
+﻿@using WebMatrix.Data;
+@{
+    //Database connectie
+    string connectionString = @"Data Source=(LocalDB)\MSSQLLocalDB;AttachDbFilename=|DataDirectory|\Database.mdf;Integrated Security=True";
+    string provider = "System.Data.SqlClient";
+    Database db = Database.OpenConnectionString(connectionString, provider);
+
+    //get the url
+    string url = Request.Url.AbsoluteUri;
+    string urlcode = url.Substring(url.Length - 7);
```

Comment:
1: Waarom haal je hier de urlcode op als je hem in je query uit een QueryString haalt?
2: Waarom haal je je urlcode uit je querystring? Daar hoort hij nooit in te staan om mee te beginnen.
3: Je haalt de urlcode incorrect op, hiermee krijg je een urlcode in de richting van 123456/3, wat geen geldige code is.
Dit is überhaupt geen geldige int, ik vraag je nog een keer: heb je je code getest voordat je het hebt gepusht? Want als je dit probeert te converteren naar een int zou je een conversion error moeten krijgen.
4: Waarom staat dit in een cshtml bestand en niet in een class? Het lijkt er niet op dat je enigzins iets met html doet en enkel C# code uitvoert dus is deze pagina compleet nutteloos om in cshtml te zetten en zal dit hoogstwaarscheinlijk voor onvoorziene problemen zorgen later.